src/Plugin/rest/resource/ContractCancelResource.php

```diff
@@ -285,9 +287,29 @@ public function post($data) {
 
       $body = '';
 
-      if (in_array($data['client'], ['norma', 'kaufland'])) {
-        foreach ($data as $data_key => $data_value) {
-          $body .= sprintf('<p>%s: %s</p>' . PHP_EOL, $data_key, $data_value);
+      $address = implode(' ', [$data['street'], $data['street number'], $data['zipcode'], $data['city']]);
+
+      if ($address) {
+        $body .= sprintf('<p>%s: %s</p>' . PHP_EOL, $this->t('Adresse'), $address);
+      }
+
+      $translations = [
+        'first name' => $this->t('Vorname'),
+        'client' => $this->t('Marke'),
+        'last name' => $this->t('Nachname'),
+        'email adress' => $this->t('E-Mail Adresse'),
+        'customer ID' => $this->t('Kundennummer'),
+        'sim card number' => $this->t('SIM-Kartennummer'),
+        'mobile phone number' => $this->t('Mobilnummer'),
+        'date of termination' => $this->t('Kündigungsdatum'),
+        'ordinary termination' => $this->t('Ordentliche Kündigung'),
+        'extraordinary termination' => $this->t('Außerordentliche Kündigung'),
+        'iban' => $this->t('Iban'),
+      ];
```

Comment:
What is the point of using t() on translated strings?
Only English language can be used inside t() and translations can be added through PO file.
https://www.drupal.org/community/contributor-guide/reference-information/localize-drupal-org/working-with-offline/po-and-pot-files#s-custom-module-translation

Reply:
Done!